src/app/(core)/v/[slug]/player.tsx

```diff
@@ -86,18 +86,33 @@ export const VodPlayer = (props: { id: string; vod: VOD }) => {
     ...props.vod.markers,
   ];
 
-  const [offset, setOffset] = useState(0);
-
+  const [offset, setOffset] = useState<{presentational: string, totalSeconds: number}>({
+    presentational: '0',
+    totalSeconds: 0
+  });
   const csv = mockedMarkers.map((marker, id) => {
     const endTime =
       (mockedMarkers[id + 1]?.position_seconds ??
-        (videoDuration as duration.Duration)?.asSeconds?.()) - offset;
+        (videoDuration as duration.Duration)?.asSeconds?.()) - offset.totalSeconds;
 
-    const startTime = Math.max(marker.position_seconds - offset, 0);
+    const startTime = Math.max(marker.position_seconds - offset.totalSeconds, 0);
 
     return `${startTime},${endTime},${marker.description.replace(",", "")}`;
   });
 
+  function parseOffsetValue(value: string): number | undefined {
+    // if there are no colons, assume its seconds
+    if(/^\d+$/.test(value)) return parseInt(value, 0);
```

Comment:
`parseInt` should prob have a radix of 10. as per [browser standards](https://262.ecma-international.org/13.0/#sec-parseint-string-radix), adding a 0 just let's the browser know you want the default radix (which, according to this is supposed to be 10 if the browser is ECMA compliant but might as well just put the expected value)

```suggestion
    if(/^\d+$/.test(value)) return parseInt(value, 10);
```